deno/src/utils/BroadcastMethods.ts

```diff
@@ -40,7 +40,12 @@ export class BroadcastMethods<T extends Record<string, Function>> {
         const result = await methods[fn](...args)
         chan.postMessage({ result, requestId })
       } catch (error) {
-        chan.postMessage({ error, requestId })
+        console.error('Can\'t post results, trying to send the error', error)
```

Comment:
What is the specific error something like 
```suggestion
        if (error?.message?.includes('{{SOME_SPECIFIC_ERROR}}')) { 
          console.error('Non-fatal error detected {{WHY_THIS_ERROR_HAPPENED}}')
        } else {
          throw error
        }
```

Reply:
Oh got it, I'll check and see what is the actual error.

Reply:
Ok the exception is because the websocket i'm trying to send a message to has already closed. Fumbling with this it seems that there are also other uncaught exceptions all regarding this kind of issues (if someone leaves and the game is still open it will still fail).

I'm still not entirely sure on the reproduction steps but given it's not the only point where the disconnection can cause unhandled exceptions i think it might be better to do a complete check in another issue (i can do it myself but i might need your help to better understand the backend code without the need to read the whole codebase).

Let me know how do you want to handle this, if you prefer i might add an error similar to what you've proposed in the meantime.

Reply:
Does it throw this error if you connect two tabs to the game and one tab closes ?

Reply:
It's kinda strange because that's exactly the other error i was talking about: if you enter with two person and quit with one it will throw an unhandled error but only at the end of the game not during the "typing events".